redirect-robots.php

```diff
@@ -278,12 +278,12 @@ function deny( ) {
 }
 
 /**
- * check for preview & verify
+ * check for staging machines (for example: preview or verify)
  */
 $headers = function_exists('apache_request_headers') ? apache_request_headers() : array();
 
 $isdeny = !empty( $headers[ "X-Staging" ] ) &&
-	preg_match("@^(sandbox|preview$|verify$)@", $headers["X-Staging"])
+	preg_match("@^(sandbox|preview$|verify$|showcase$)@", $headers["X-Staging"])
```

Comment:
Can we actually disable indexing of all staging environments? This list is simply getting harder and harder to maintain ;)